x9.md

```diff
@@ -0,0 +1,57 @@
+# Status Account Creation and Maintenance Specification
+
+## Summary
+
+The core concept of an account in Status is a set of cryptographic keypairs.  Namely, the combination of the following:
+1. a whisper chat identity keypair
+1. a set of cryptocurrency wallet keypairs
+
+Everything else associated with the contact is either verified or derived from the above items, including:
+- Ethereum address (future verification, currently the same base keypair)
+- 3 word mnemonic name
+- identicon
+- message signatures
+
+## 1 Initial Key Generation
+### 1.1 Public/Private Keypairs 
+- An ECDSA (secp256k1 curve) public/private keypair MUST be generated via a [BIP43](https://github.com/bitcoin/bips/blob/master/bip-0043.mediawiki) derived path from a [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) mnemonic seed phrase.
+- The default paths are defined as such:
+    - Whisper Chat Key ($IK$): `m/43'/60'/1581'/0'/0`  (post Multiaccount integration)
```

Comment:
This is hard to reason about tbh (time dependent specs), lets make sure we simplify this. The goal of this should be v1 specs, and there multiaccount will be used.

Also, should we have a separate doc for multiaccount?

Reply:
I don't think we should separate for multi-account, unless you want to separate the wallet spec from the chat spec.  In my eyes, the account encompasses all a user is and needs to do to participate in the network, at least in terms of keys.